egs/wsj/s5/steps/make_phone_graph.sh

```diff
@@ -92,17 +92,14 @@ fi
 
 if [ $stage -le 3 ]; then
   echo "$0: creating G_phones.fst from ARPA"
-  gunzip -c $dir/phone_graph/arpa_noug.gz | arpa2fst - - | fstprint | \
-    utils/eps2disambig.pl | utils/s2eps.pl | \
-    awk '{if (NF < 5 || $5 < 100.0) { print; }}' | \
-    fstcompile --isymbols=$lang/phones.txt --osymbols=$lang/phones.txt \
-       --keep_isymbols=false --keep_osymbols=false | \
-    fstconnect | \
-    fstrmepsilon > $dir/phone_graph/G_phones.fst
-   fstisstochastic $dir/phone_graph/G_phones.fst  || echo "[info]: G_phones not stochastic."
+  gunzip -c $dir/phone_graph/arpa_noug.gz | \
+    arpa2fst --disambig-symbol=#0 --read-symbol-table=$lang/phones.txt - - | \
+    fstprint | awk '{if (NF < 5 || $5 < 100.0) { print; }}' | fstcompile | \
```

Comment:
How long a usage line is acceptable?

Would this statement be more clear:

> Store symbol table with FST. Symbols are always stored if symbol tables are neither read or written otherwise

This complication is intended to keep arpa2fst backward-compatible. The old behavior of just `arpa2fst input | fstprint` is to print the FST with symbols. I could have set the default to true always, but that would be a wholesale inconvenience in the new invocation pattern that would have to carry --keep_symbols=false over every call. Hence I took the tradeoff of a more complex default. Arguably, this makes some intuitive sense: if an ARPA file is read, but no words.txt is provided and the symbols are not saved to a separate file, force save the symbols to FST, otherwise they will be lost and the FST will be useless. In another program I would just print an error message if an invalid combination of switches is provided, but in this case it does not cut it.